users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -271,11 +299,131 @@ def run_scf_baseline():
+        "conformer_type": "wei",
+        "specaug_old": {"max_feature": 15},
+        "extra_args": {
+            "accum_grad_multiple_step": 2,
+            "conv_pad_seq_len_to_power": 1.5,
+        },
+    }
+
+    common_feature_args = {
+        "class": "ScfNetwork",
+        "size_tf": 256 // 2,
+        "stride_tf": 10 // 2,
+        "preemphasis": 0.97,
+        "wave_norm": True,
+    }
+
+    common_lr_args = {
+        "peak_lr": 4e-4,
+        "start_lr": 1.325e-05,
+        "end_lr": 1e-5,
+        "increase_epochs": 180,
+        "decrease_epochs": 180,
+        "final_epochs": 0,
+    }
+
+    common_report_args = {
+        "architecture": "conf-wei",
+        "lr": "wei_peak_4e-4_e450_cycle360",
+        "specaug": "wei_adapt_80dim",
+        "wave_norm": "True",
+        "preemphasis": "0.97",
+    }
+
+    nn_args, report_args_collection = get_nn_args_baseline(
+        nn_base_args={
+            "epoch_256_freezing": {
+                "returnn_args": {
+                    "staged_opts": {
+                        256: "freeze_features",
+                    },
+                    **common_returnn_args,
+                },
+                "feature_args": common_feature_args,
+                "lr_args": common_lr_args,
+                "report_args": common_report_args,
+            },
+            "fixed_features": {
+                "returnn_args": {
+                    **common_returnn_args,
+                    "staged_opts": {
+                        1: "freeze_features",
+                    },
+                    "extra_args": {
+                        **common_returnn_args["extra_args"],
+                        "preload_from_files": {
+                            "existing-model": {
+                                "filename": nn_system.train_jobs[
+                                    "conformer_bs2x5k_scf_baseline_preemphasis97_wn"
+                                ].out_checkpoints[400],
+                            "init_for_train": True,
+                            "prefix": "features",
+                            "var_name_mapping": {
+                                "/conv_h_filter/conv_h_filter": "features/conv_h_filter/conv_h_filter",
+                                "/conv_l/W": "features/conv_l/W",
+                                "/conv_l_act/bias": "features/conv_l_act/bias",
+                                "/conv_l_act/scale": "features/conv_l_act/scale",
+                                "/wave_norm/bias": "features/wave_norm/bias",
+                                "/wave_norm/scale": "features/wave_norm/scale",
+                                },
+                            },
+                        },
+                    },
+                },
+                "feature_args": common_feature_args,
+                "lr_args": common_lr_args,
+                "report_args": common_report_args,
+            }
+        },
+        num_epochs=450,
+        evaluation_epochs=[350, 400, 450],
```

Comment:
```suggestion
        evaluation_epochs=[350, 390, 400, 410, 450],
```
Maybe also more recognitions here?